practice/2_python_part_2/task_read_write_2.py

```diff
@@ -22,3 +22,24 @@ def generate_words(n=20):
         words.append(word)
 
     return words
+
+def write_to_file():
```

Comment:
The same - better to split to two functions because it would be easier to test that one function storing results in utf and another one in cp1251